src/View/Helper/FlashMessenger.php

```diff
@@ -326,8 +356,11 @@ protected function getEscapeHtmlHelper()
             return $this->escapeHtmlHelper;
         }
 
-        if (null !== $this->getView() && method_exists($this->getView(), 'plugin')) {
-            $this->escapeHtmlHelper = $this->view->plugin('escapehtml');
+        $view = $this->getView();
+
+        if ($view && method_exists($view, 'plugin')) {
+            /** @var EscapeHtml|null $this->escapeHtmlHelper */
```

Comment:
This should never be null at this stage: let's also replace it with an assertion